### PR TITLE
fix: run tests on breaking_python branch

### PR DESCRIPTION
**Pull request by ojeytonwilliams**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

### CODE REVIEW

File: .github/workflows/node.js-tests.yml
Comment (naomi-lgbt) [inline .github/workflows/node.js-tests.yml:1]: Will these changes take effect if they are not in the `main` branch?

Comment (ojeytonwilliams) [reply .github/workflows/node.js-tests.yml:1]: Yep, there's already an action queued on this PR.

**Review by naomi-lgbt — APPROVED** [review]

**Comment by ojeytonwilliams** [comment]
@Sembauke does mobile care about the value of the challenge type? Beyond the fact it's a number, that is.

**Review by ShaunSHamilton — APPROVED** [review]
I am fine with adding this, but it seems unnecessary as one of the main reasons for using a feature branch was to not take up CI when we can just double-checking things once we are reviewing the `breaking_python -> main` PR.

**Comment by ojeytonwilliams** [comment]
I get that, but the CI isn't _that_ bad right now. Also, it's easier to fix bugs immediately than later down the line.

**Pull request merged**